src/functions/schema.ts

```diff
@@ -14,6 +15,12 @@ ajv._opts.defaultMeta = jsonSpecv4.id;
 // @ts-ignore
 ajv._refs['http://json-schema.org/schema'] = 'http://json-schema.org/draft-04/schema';
 
+ajv.addFormat('int32', { type: 'number', validate: validator.int32 });
+ajv.addFormat('int64', { type: 'number', validate: validator.int64 });
+ajv.addFormat('float', { type: 'number', validate: validator.float });
+ajv.addFormat('double', { type: 'number', validate: validator.double });
+ajv.addFormat('byte', { type: 'string', validate: validator.byte });
```

Comment:
I had a discussion with @philsturgeon about this just yesterday and he was against this. Did you folks change your mind?

Reply:
Ok, this approach is a bit nicer than what I had in mind when we discussed using ajv-oas. I thought we'd be passing all validation through that function, instead we're kinda teaching our generic validation logic about some of the formats OpenAPI people are likely to use, and seeing as OpenAPI and JSON Schema are used interchangeably by many its not hurting to have these available in generic JSON Schema usage anyway. 

Let's go with this, and if we get pushback about having OpenAPI-specific formats enabled for all the things then... ok we can look into adding specific validation rules for specific rulesets, or autodetecting OpenAPI, or some other thing.

Reply:
Hmm, ok. I fail to understand the difference with what was proposed last week though. It was the exact same solution.